meeko/data/residue_chem_templates.json

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
We should remember to include a description of what this data file is in documentation. I'm putting this comment here as a reminder, I'll add that to the documentation issue more explicitly and resolve this comment later.